examples/step-44/step-44.cc

```diff
@@ -31,10 +31,13 @@
 #include <deal.II/base/tensor.h>
 #include <deal.II/base/timer.h>
 #include <deal.II/base/work_stream.h>
-
 #include <deal.II/dofs/dof_renumbering.h>
 #include <deal.II/dofs/dof_tools.h>
 
+// This header gives us the functionality to store
+// data at quadrature points
+#include <deal.II/base/quadrature_point_data.h>
```

Comment:
I still think that it makes sense to keep these all together, since they're so closely related and relatively light-weight classes. But thats just my 2 cents.